src/pages/Details.js

```diff
@@ -37,6 +37,7 @@ const Details = () => {
         <div className="details-header">
           <span className="breed-name">
             {breedDetails}
+            {' '}
```

Comment:
- [ ] Great job so far I really appreciate your great job but I just want to note somthing kindly add the navbar as given in the design since I can't see in your design so please just add the navbar to make your project complete 🔥🔥🔥✨✨✨


- Your design
![image](https://user-images.githubusercontent.com/58553711/210679458-8a4cb43c-bdf6-433a-b453-f61df2def0ed.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/210679537-88f41609-36f8-443d-bf84-33deb2abeb61.png)